src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Inputs/FileInput/BitFileInputDemo.razor.scss

```
@@ -1,21 +1,38 @@
-    background-color: #f5f5f5;
-    border-radius: 4px;
+.example-box {
+    gap: 1rem;
+    display: flex;
+    flex-direction: column;
+}
 
-    strong {
-        display: block;
-        margin-bottom: 8px;
+::deep {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)